data/quickstarts/create-jupyter-notebook-quickstart.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   appName: jupyterhub
   durationMinutes: 5
   icon: 'images/jupyterhub.svg'
-  description: This quick start will walk you through creating a Jupyter notebook.
+  description: Create a Jupyter notebook in the Red Hat® OpenShift® Dedicated environment with JupyterHub. 
```

Comment:
I would change to: "Create a Jupyter notebook in JupyterLab."  The rationale is that not all customers will use OpenShift Dedicated - some will use ROSA. Also, don't want to add more JupyterHub references since we're moving away from it.

Reply:
LGTM

Reply:
Changed